src/mixins/createFormFieldMixin.ts

```diff
@@ -76,52 +102,38 @@ const createFormMixin: FormMixinFactory = compose({
 					this.setState({ value: stringToValue(event.value) });
 				}
 			}
-		}
-	}, (instance: FormField<any>, options: FormFieldMixinOptions<any, FormFieldMixinState<any>>) => {
-		if (options) {
-			const { type } = options;
-			if (type) {
-				instance.type = type;
+		},
+
+		nodeAttributes: [ function (this: FormFieldMixin<any, FormFieldMixinState<any>>): VNodeProperties {
```

Comment:
what is the standard for this, we have some examples where `function` is new lined and some inline. I think it reads slightly better new lined by its very subjective.